modules/conversion-center/assets/scss/frontend.scss

```diff
@@ -13,3 +13,5 @@
 @import "widgets/contact-buttons-var-6";
 @import "widgets/contact-buttons-var-7";
 @import "widgets/contact-buttons-var-8";
+
+@import "editor/contact-buttons-editor.scss";
```

Comment:
@mserino I think the editor imports are here: `elementor/assets/dev/scss/editor/editor.scss`
We don't need this SCSS in the frontend file.

Reply:
ah, thanks!

Reply:
@louiswol94 okay, pushed a fix for this. It needed to be imported in the `preview`